BetaCameras/RealSense2/RealSense2.cs

```diff
@@ -1416,6 +1418,8 @@ protected override void DisconnectImpl()
             try
             {
                 StopPipeline();
+                _intrinsics.Clear();
```

Comment:
The cache will not get cleared if `StopPipeline` fails. I would clear it outside the try-catch block.

Reply:
Oh, super obvious now that you point it out. But somehow I missed that.